ArchiSteamFarm/Steam/Integration/ArchiWebHandler.cs

```diff
@@ -2756,5 +2879,12 @@ private enum ESteamApiKeyState : byte {
 			NotRegisteredYet,
 			AccessDenied
 		}
+		private enum ESteamAccessTokenState : byte {
```

Comment:
This is completely not needed, you're not dealing here with any special logic that I had to deal with using API keys, you don't need two dedicated functions for it either.

```csharp
			Uri request = new(SteamStoreURL, "/pointssummary/ajaxgetasyncconfig");

			ObjectResponse<AccessTokenResponse>? response = await UrlGetToJsonObjectWithSession<AccessTokenResponse>(request).ConfigureAwait(false);

			if (response == null) {
				return (false, null);
			}

			return (true, response.Content.AccessTokenData.Webapi_token);
```

That's all your resolve function should be doing.